### PR TITLE
Date time format

### PR DESCRIPTION
**Pull request by AbijithS-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes

made all date and time configurable and made default format as dd-mm-yyyy
# How has the change been tested?

# Screenshots (if applicable)

image

### CODE REVIEW

File: forms-flow-web-root-config/docker-compose.yml [hunk 1]
```diff
@@ -43,6 +43,8 @@ services:
       - REACT_APP_ENABLE_DASHBOARDS_MODULE=${ENABLE_DASHBOARDS_MODULE:-true}
       - REACT_APP_ENABLE_PROCESSES_MODULE=${ENABLE_PROCESSES_MODULE:-true}
       - REACT_APP_ENABLE_APPLICATIONS_MODULE=${ENABLE_APPLICATIONS_MODULE:-true}
+      - REACT_APP_DATE_FORMAT=${DATE_FORMAT}
```
Comment (abhilash-aot) [inline forms-flow-web-root-config/docker-compose.yml:46]: Add to sample.env , config template/ sample and add in. Change log too about the env

Comment (AbijithS-aot) [reply forms-flow-web-root-config/docker-compose.yml:46]: ok

**Review by abilpraju-aot — COMMENTED** [review]
ci breaking unit test cases check

**Comment by AbijithS-aot** [comment]
For test cases in web helperservice will be available only in the run time so it cannot be expected in the screen during test run which will get undefined since it is available only in the runtime. So removing those cases from the web

**Pull request closed without merging**